src/Files.Shared/Helpers/PathHelpers.cs

```diff
@@ -17,77 +17,28 @@ public static string Combine(string folder, string name)
 			return folder.Contains('/') ? Path.Combine(folder, name).Replace('\\', '/') : Path.Combine(folder, name);
 		}
 
-		public static string FormatName(string path)
-		{
-			string fileName;
-			string rootPath = Path.GetPathRoot(path) ?? string.Empty;
-
-			if (rootPath == path && 
-					( path.StartsWith(@"\\") ||
-					  (new DriveInfo(path).DriveType == System.IO.DriveType.Network)
-					)
-			   )
-			{
-				// Network Share path
-				fileName = path.Substring(path.LastIndexOf(@"\", StringComparison.Ordinal) + 1);
-			}
-			else if (rootPath == path)
-			{
-				// Drive path
-				fileName = path;
-			}
-			else
-			{
-				// Standard file name
-				fileName = Path.GetFileName(path);
-			}
-
-			// Check for link file name
-			if (FileExtensionHelpers.IsShortcutOrUrlFile(fileName))
-				fileName = fileName.Remove(fileName.Length - 4);
-
-			return fileName;
-		}
-
-		/// <summary>
-		/// Determines whether the <paramref name="path"/> points to any special system folders.
-		/// </summary>
-		/// <remarks>
-		///	The term "Special folder" refers to any folders that may be natively supported by a certain platform (e.g. Libraries).
-		/// </remarks>
-		/// <param name="path">The path to a folder to check.</param>
-		/// <returns>If the path points to a special folder, returns true; otherwise false.</returns>
-		public static bool IsSpecialFolder(string path)
-		{
-			foreach (Environment.SpecialFolder specialFolder in Enum.GetValues(typeof(Environment.SpecialFolder)))
```

Comment:
This was AOT unsafe but referenced from nowhere, was safe to remove.

Reply:
Do you know what it was added for?

Reply:
Was started referencing in #13149 but removed in #16900

Reply:
FYI @d2dyno1